src/ccs/cccs.rs

```diff
@@ -158,17 +158,14 @@ mod tests {
     vecs.iter().map(Vec::as_slice).collect()
   }
 
-  /// Do some sanity checks on q(x). It's a multivariable polynomial and it should evaluate to zero inside the
-  /// hypercube, but to not-zero outside the hypercube.
```

Comment:
Why is the comment removed?

Reply:
It has been moved some lines down, on top of the generic test invoker.

Reply:
Just saw yes